src/Common/src/Interop/Windows/HttpApi/Interop.HttpApi.cs

```diff
@@ -786,75 +786,70 @@ internal static unsafe string GetVerb(HTTP_REQUEST* request)
             return GetVerb(request, 0);
         }
 
-        internal static unsafe string GetVerb(byte[] memoryBlob, IntPtr originalAddress)
+        internal static unsafe string GetVerb(IntPtr memoryBlob, IntPtr originalAddress)
         {
-            fixed (byte* pMemoryBlob = memoryBlob)
```

Comment:
The vast majority of the line changes in this file are the result of removing these fixed statements, and thus changing the indentation for large blocks of code.

Reply:
> The vast majority of the line changes in this file are the result of removing these fixed statements, and thus changing the indentation for large blocks of code

FYI, if you append ?w=1 to the GitHub url, it'll ignore whitespace differences:
https://github.com/dotnet/corefx/pull/25563/files?w=1

Reply:
Good to know!